### DISCUSSION TITLE
Database Setup

### DISCUSSION THREAD

**Discussion by AssadRajab** [Xeno69/Domination#631] [Q&A]
Hi, The [database setup guide](https://github.com/Xeno69/Domination/wiki/Database-setup) says: _don’t forget to copy the tbbmalloc dll files, both, from your extDB3 download to the A3 server folder_. However, I don't see such a folder on the server. I am using [Faster](https://github.com/Foxlider/FASTER) and I don't see such a folder. Any idea on that?
![here how my folder looks like ](https://github.com/Xeno69/Domination/assets/26144219/a42cb795-2209-42c6-b9cf-6ab30bec95e6)

**Comment by Xeno69** [comment]
Answered on Discord
